packages/react-scripts/layout/views/partials/dynatrace.ejs

```diff
@@ -0,0 +1,80 @@
+    Je}else Fd=Gf}if(Fd)for(var Cc in Fd)Object.prototype.hasOwnProperty.call(Fd,Cc)&&(mg=Cc,xc[mg]=Fd[mg]);var uc=yb();try{var uf=(Fd=lg)&&Fd.getItem(uc);if(uf){var Ae=Ha(uf),Nd=vb(Ae.config||""),cf=xc.lastModification||"0",df=U((Nd.lastModification||Ae.lastModification||"0").substring(0,13)),ug="string"===typeof cf?U(cf.substring(0,13)):cf;if(!cf||df>=ug)if(Nd.csu=Ae.name||I("csu"),Nd.featureHash=Ae.featureHash||I("featureHash"),Nd.agentUri&&Cb(Nd),Qb(Nd,!0),Ab(Nd),jb(Nd),df>(xc.lastModification||0)){var Ud=
+    Oa("auto"),gj=Oa("legacy");xc=Ba(Nd);xc.auto=Ud?"1":"0";xc.legacy=gj?"1":"0"}}}catch(Xb){}Qb(xc);try{var Hi=xc.ign;if(Hi&&(new RegExp(Hi)).test(Bb.location.href)){document.dT_=Bb.dT_=void 0;var se=!1;break a}}catch(Xb){}if(mc()){var oi=navigator.userAgent,Yd=oi.lastIndexOf("RuxitSynthetic");if(-1===Yd)var $f={};else{var qf=oi.substring(Yd+14);if(-1===ra(qf," c"))$f={};else{uc={};uf=0;for(var rf=qf.split(" ");uf<rf.length;uf++){var ag=rf[uf];if("c"===ag.charAt(0)){var yf=ag.substring(1),Ch=yf.indexOf("="),
+    Pe=yf.substring(0,Ch),Dh=yf.substring(Ch+1);Pe&&Dh&&(uc[Pe]=Dh)}}$f=uc}}qf=void 0;for(qf in $f)Object.prototype.hasOwnProperty.call($f,qf)&&$f[qf]&&(xc[qf]=$f[qf]);Ba(xc)}se=!0}if(!se)throw Error("Error during config initialization");Qd();try{Ye=Bb.dT_.disabled||!!oa("dtDisabled")}catch(Xb){}var Eg;if(!(Eg=I("agentLocation")))a:{var Ug=I("agentUri");if(Ug||document.currentScript){var D=Ug||document.currentScript.src;if(D){se=D;var P=-1===ra(se,"_bs")&&-1===ra(se,"_bootstrap")&&-1===ra(se,"_complete")?
+    1:2,da=D.lastIndexOf("/");for(se=0;se<P&&-1!==da;se++)D=D.substring(0,da),da=D.lastIndexOf("/");Eg=D;break a}}var xa=location.pathname;Eg=xa.substring(0,xa.lastIndexOf("/"))}Ff=Eg;Tg=I("csu")||"ruxitagentjs";"true"===ba("dtUseDebugAgent")&&0>Tg.indexOf("dbg")&&(Tg=I("debugName")||Tg+"dbg");if(!Oa("auto")&&!Oa("legacy")&&!Ye){var Qa=I("agentUri")||Of(I("featureHash")),hb;if(!(hb=Oa("async")||"complete"===document.readyState)){var Gb=Bb.navigator.userAgent,Jb=Gb.indexOf("MSIE ");hb=0<Jb?9>=parseInt(Gb.substring(Jb+
+    5,Gb.indexOf(".",Jb)),10):!1}if(hb)xf(Qa,Oa("async"),void 0,void 0,"dtjsagent");else{var Rb="".concat("dtjsagent","dw");document.write('<script id="'.concat(Rb,'" type="text/javascript" src="').concat(Qa,'">\x3c/script>'));document.getElementById(Rb)||xf(Qa,Oa("async"),void 0,void 0,"dtjsagent")}}ba("dtCookie")&&ka("cooO",!0);pf();ka("pVO",!!hd("dt-pVO"));ad(od);Df=18E5;Mf=Za("hvt")||216E5;y(Xe,[1]);Rg=[];Xf="dtCookie dtLatC rxvt dtAdk dtAdkSettings dtPC".split(" ");if(Oa("cg"))try{Wc(Object.getOwnPropertyDescriptor(Document.prototype,
+    "cookie")||Object.getOwnPropertyDescriptor(HTMLDocument.prototype,"cookie"))}catch(Xb){}}catch(Xb){try{delete Bb.dT_}catch(Dc){Bb.dT_=void 0}La()&&Bb.console.log("JsAgent initCode initialization failed!",Xb)}})(!1)})()})();
+    </script>
+<% } else if (dynatraceFlag.treatment.isOn) { %>
```

Comment:
I think it's probably not the best practice to use a multivariant flag with the treatment value of 'on'.   I think we should name this treatment something more specific.

Reply:
ok, I have renamed the treatment to "global-cdn", updated both PRs and updated the Split config for all environments.